widgets/Liveshopping/index.jsx

```diff
@@ -61,11 +61,11 @@ const createProductSliderItem = ({
 
   return (
     <div key={id} className={styles.card} data-test-id="liveShoppingWidget">
-      <Card>
+      <Card className={styles.cardInnerBox}>
         <Link href={`${ITEM_PATH}/${bin2hex(id)}`}>
           <Grid>
             <Grid.Item className={styles.imagePane}>
-              <ProductImage src={featuredImageUrl} alt={name} />
+              <ProductImage styleUpdate={{container: styles.updateImageContainer, glowContainer: styles.updateImageGlowContainer }} src={featuredImageUrl} alt={name} />
```

Comment:
Since the card container has a border radius defined already and overflow:hidden, we actually don't need to make any further adjustments. The corners will be "cropped" by the card's css.